src/object.js

```diff
@@ -146,11 +161,40 @@ ReflectionObject.prototype.onRemove = function onRemove(parent) {
 ReflectionObject.prototype.resolve = function resolve() {
     if (this.resolved)
         return this;
+    if (this instanceof Root || this.parent && this.parent.resolved)
```

Comment:
How does this work?  It looks to me like this.parent.resolved will only be true for the root.  So for any nested objects `_resolveFeatures` will never get called?